node/src/Commands.ts

```diff
@@ -2441,9 +2441,9 @@ export function createFunctionRestore(
  */
 export type BitOffsetOptions = {
     /** The starting offset index. */
-    start: number;
+    start?: number;
```

Comment:
isn't start required?  
```suggestion
    start: number;
```

Reply:
no it is optional. only key and bit are required

Reply:
Changing `start` to required. See this comment thread on the python PR for context: https://github.com/valkey-io/valkey-glide/pull/2256#discussion_r1774178699